src/app/features/profile/store/profile.effects.ts

```diff
@@ -12,10 +13,20 @@ export class ProfileEffects {
         this.action$.pipe(
             ofType(getProfilePending),
             switchMap(() => this.usersService.getProfile()
-            .pipe(map((profile: IProfile) => getProfileSuccess({ profile }))))
+            .pipe(
+                tap((value) =>
+                    value
+                        ? this.notificationService.showNotification('Get Profile Success!', 3000, 'yellow')
+                        : null),
+                map((profile: IProfile) => getProfileSuccess({ profile }))
```

Comment:
I kinda confused, you are expecting that `value` may be negative above but still are sending `getProfileSuccess` even in that case.
That means you won't show notification but still update store with negative results which may lead to an error
Or there is no possibility of negative value and this check is excess?